src/Entity/User.php

```diff
@@ -1129,7 +1129,7 @@ public function getFirstUserRelay(): ?UserCentre
 
     public static function createUserRelay(User $user, Centre $relay): UserCentre
     {
-        $userRelay = $user->isBeneficiaire() ? new BeneficiaireCentre() : new MembreCentre();
+        $userRelay = $user->isBeneficiaire() ? new BeneficiaireCentre() : new MembreCentre(true, false);
```

Comment:
Why `true` is not the default for first param ? 
You can also have `new MembreCentre(true)`

Reply:
Because if the default value in MembreCentre constructor is true for beneficiary management, it will be the default during admin creation too